mbq/ranch/__init__.py

```diff
@@ -8,5 +8,6 @@
     __version__,
 )
 
+from .killswitch import create_killswitch_task_class  # noqa
```

Comment:
OK I'm just gonna say it. We don't need to talk about it.

But how nice would it be if this was `from . import killswitch` and then you'd use it with

```python
from mbq import ranch
ranch.killswitch.create_task_class(foo.variation)
```

Reply:
I like this, made the change.